### ISSUE TITLE
1.Module parse failed: 'import' and 'export' may appear only with 'sourceType: module' (1:0),2.Error: [BABEL] Unknown option: .visitor. 

### ISSUE THREAD

**Issue by masx200** [pmmmwh/react-refresh-webpack-plugin#451]
1.Module parse failed: 'import' and 'export' may appear only with 'sourceType: module' (1:0),

2.Error: [BABEL] Unknown option: .visitor.
```shell
webpack serve  --config ./release/config/webpack.config.js  --mode=development --port=11111
```
```txt
ERROR in ./src/index.jsx

Module build failed (from ./node_modules/.pnpm/babel-loader@8.2.2_bdbf93c44932bbce0b7b7ee8ec67053a/node_modules/babel-loader/lib/index.js):

Error: [BABEL] D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\src\index.jsx: Unknown option: .visitor. Check out https://babeljs.io/docs/en/babel-core/#options for more information about options.

    at throwUnknownError (D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\node_modules\.pnpm\@babel+core@7.14.6\node_modules\@babel\core\lib\config\validation\options.js:133:27)

    at D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\node_modules\.pnpm\@babel+core@7.14.6\node_modules\@babel\core\lib\config\validation\options.js:118:5

    at Array.forEach (<anonymous>)

    at validateNested (D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\node_modules\.pnpm\@babel+core@7.14.6\node_modules\@babel\core\lib\config\validation\options.js:94:21)

    at validate (D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\node_modules\.pnpm\@babel+core@7.14.6\node_modules\@babel\core\lib\config\validation\options.js:85:10)

    at D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\node_modules\.pnpm\@babel+core@7.14.6\node_modules\@babel\core\lib\config\full.js:324:36

    at cachedFunction (D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\node_modules\.pnpm\@babel+core@7.14.6\node_modules\@babel\core\lib\config\caching.js:60:27)

    at cachedFunction.next (<anonymous>)

    at evaluateSync (D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\node_modules\.pnpm\gensync@1.0.0-beta.2\node_modules\gensync\index.js:251:28)

    at sync (D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\node_modules\.pnpm\gensync@1.0.0-beta.2\node_modules\gensync\index.js:89:14)

    at loadPresetDescriptor (D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\node_modules\.pnpm\@babel+core@7.14.6\node_modules\@babel\core\lib\config\full.js:313:18)

    at loadPresetDescriptor.next (<anonymous>)

    at recursePresetDescriptors (D:\Documents\GitHub\webpack-react-vue-spa-awesome-config\node_modules\.pnpm\@babel+core@7.14.6\node_modules\@babel\core\lib\config\full.js:114:30)

    at recursePresetDescriptors.next (<anonymous>)

nfig\full.js:191:21

    at Generator.next (<anonymous>)

ERROR in ./node_modules/.pnpm/@pmmmwh+react-refresh-webpack-plugin@0.5.0-rc.2_90444ac847f36353930b8f2320929e70/node_modules/@pmmmwh/react-refresh-webpack-plugin/client/utils/patchUrl.cjs 1:0

Module parse failed: 'import' and 'export' may appear only with 'sourceType: module' (1:0)

File was processed with these loaders:

 * ./node_modules/.pnpm/file-loader@6.2.0_webpack@5.44.0/node_modules/file-loader/dist/cjs.js

You may need an additional loader to handle the result of these loaders.

> export default __webpack_public_path__ + "node_modules/.pnpm/@pmmmwh+react-refresh-webpack-plugin@0.5.0-rc.2_90444ac847f36353930b8f2320929e70/node_modules/@pmmmwh/react-refresh-webpack-plugin/client/utils/patchUrl.cjs";

 @ ./node_modules/.pnpm/@pmmmwh+react-refresh-webpack-plugin@0.5.0-rc.2_90444ac847f36353930b8f2320929e70/node_modules/@pmmmwh/react-refresh-webpack-plugin/client/ErrorOverlayEntry.js?sockHost=0.0.0.0&sockPort=11111&sockProtocol=http 4:0-53 81:2-19

webpack 5.44.0 compiled with 2 errors in 2013 ms
```
```js
("use strict");
 const ReactRefreshTypeScript = require("react-refresh-typescript");
const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
 const ReactRefreshWebpackPlugin = require("@pmmmwh/react-refresh-webpack-plugin");
const fs = require("fs");
const CopyPlugin = require("copy-webpack-plugin");
const WorkboxWebpackPlugin = require("workbox-webpack-plugin");
const safePostCssParser = require("postcss-safe-parser");
const OptimizeCSSAssetsPlugin = require("optimize-css-assets-webpack-plugin");
const TerserPlugin = require("terser-webpack-plugin");
const { VueLoaderPlugin } = require("vue-loader");
const MiniCssExtractPlugin = require("mini-css-extract-plugin");
const path = require("path");
const HtmlWebpackPlugin = require("html-webpack-plugin");
const webpack = require("webpack");
const postcssNormalize = require("postcss-normalize");

const PreloadWebpackPlugin = require("@vue/preload-webpack-plugin");

const { CleanWebpackPlugin } = require("clean-webpack-plugin");
function parseargs(args) {
    const 参数obj = {};
    args.filter((s) => s.startsWith("--"))
        .map((s) => /--(?<key>.+?)=(?<value>.+)/g.exec(s))
        .forEach((execArray) => {
            var _a, _b, _c;
            const groups =
                (_a = execArray) === null || _a === void 0 ? void 0 : _a.groups;
            const key =
                (_b = groups) === null || _b === void 0 ? void 0 : _b.key;
            const value =
                (_c = groups) === null || _c === void 0 ? void 0 : _c.value;
            if (key && value) {
                参数obj[key] = value;
            }
        });
    return 参数obj;
}
console.log("输入的参数:");
console.log(JSON.stringify(process.argv, null, 4));
const 参数object = parseargs(process.argv);
console.log("解析的参数:");
console.log(JSON.stringify(参数object, null, 4));
const 解析参数publicpath = 参数object["output-public-path"];
process.env.NODE_ENV =
    参数object["mode"] === "production"
        ? "production"
        : 参数object["mode"] === "development"
        ? "development"
        : process.env.NODE_ENV;
const isDevelopment = process.env.NODE_ENV !== "production";
const defaultport = 10000;
const port =
    参数object["port"] || defaultport + parseInt(String(10000 * Math.random()));
console.log(`\nwebpack config filename : ${__filename}\n`);
console.log(`\nworking directory : ${process.cwd()}\n`);
var __dirname = process.cwd();
console.log(`\nwebpack mode : ${process.env.NODE_ENV} \n`);
const webpackEnv = process.env.NODE_ENV;
const isEnvDevelopment = "development" === webpackEnv;
const shouldUseSourceMap = isEnvDevelopment;
const isEnvProduction = "production" === webpackEnv;
process.env.BABEL_ENV = process.env.NODE_ENV;
let publicPath = isEnvProduction ? "./" : "/";
if ("production" === process.env.NODE_ENV) {
    if (解析参数publicpath) {
        if (解析参数publicpath.length) {
            console.log(`  output-public-path  :  ${解析参数publicpath}`);
            console.log("\n");
            publicPath = 解析参数publicpath;
        }
    }
}
const isservemode = process.argv.includes("serve");
if (isEnvDevelopment & isservemode) {
    console.log("open in browser: http://localhost:" + port);
}
module.exports = {
    stats: {
        children: true,
    },
    target: isEnvDevelopment
        ? "web"
        : "browserslist:" + [">0.2%", "not dead", "not op_mini all"].join(","),
    resolve: {
        extensions: [".ts", ".js", ".tsx", ".jsx", ".cjs", ".mjs"],

        alias: { "@": path.join(__dirname, "src") },
    },
    devServer: {
        host: "0.0.0.0",
        compress: true,
        historyApiFallback: true,
        contentBase: path.resolve(__dirname, "./dist"),
        hot: !0,
        port,
        inline: !0,
        watchContentBase: !0,
    },
    devtool: isEnvDevelopment ? "eval-cheap-module-source-map" : false,
    mode: process.env.NODE_ENV,
    entry: [
        fs.existsSync(path.join(__dirname, "src", "index.tsx"))
            ? path.join(__dirname, "src", "index.tsx")
            : fs.existsSync(path.join(__dirname, "src", "index.ts"))
            ? path.join(__dirname, "src", "index.ts")
            : fs.existsSync(path.join(__dirname, "src", "index.jsx"))
            ? path.join(__dirname, "src", "index.jsx")
            : path.join(__dirname, "src", "index.js"),
    ].filter(Boolean),
    output: {
        publicPath,
        globalObject: `( Function('return this')())`,
        filename: isEnvDevelopment
            ? "[name].[fullhash].js"
            : "[name].[contenthash].js",
        path: path.join(__dirname, "dist"),
        chunkFilename: isEnvDevelopment
            ? "[name].[fullhash].js"
            : "[name].[contenthash].js",
    },
    module: {
        strictExportPresence: !0,
        rules: [
            {
                test: /\.(js|mjs|jsx|ts|tsx)$/,
                type: "javascript/auto",
                loader: require.resolve("babel-loader"),
                options: {
                    sourceMaps: shouldUseSourceMap,
                    plugins: [
                        [
                            require.resolve(
                                "@babel/plugin-proposal-decorators"
                            ),
                            { legacy: true },
                        ],
                        [
                            "@babel/plugin-proposal-class-properties",
                            { loose: true },
                        ],
                    ].filter(Boolean),
                    presets: [require.resolve("babel-preset-react-app")],
                    babelrc: false,
                    configFile: false,
                    cacheDirectory: !0,
                    cacheCompression: isEnvProduction,
                    compact: isEnvProduction,
                },
                include: [path.resolve(__dirname)],
                exclude: [],
            },

            {
                test: /\.(less)$/,
                use: [
                    isEnvDevelopment
                        ? {
                              loader: require.resolve("style-loader"),
                              options: {},
                          }
                        : { loader: MiniCssExtractPlugin.loader },
                    {
                        loader: require.resolve("css-loader"),
                        options: { sourceMap: shouldUseSourceMap },
                    },
                    {
                        loader: require.resolve("postcss-loader"),
                        options: {
                            postcssOptions: {
                                plugins: [
                                    require("postcss-flexbugs-fixes"),
                                    require("postcss-preset-env")({
                                        autoprefixer: { flexbox: "no-2009" },
                                        stage: 3,
                                    }),
                                    postcssNormalize(),
                                ],
                            },
                            sourceMap: shouldUseSourceMap,
                        },
                    },
                    {
                        loader: require.resolve("less-loader"),
                        options: { sourceMap: shouldUseSourceMap },
                    },
                ],
            },
            {
                test: /\.worker\.js$/,
                loader: require.resolve("worker-loader"),
                options: { inline: "no-fallback" },
            },
            {
                test: /\.vue$/,
                loader: require.resolve("vue-loader"),
            },
            {
                test: /\.(css|sass|scss)$/,
                use: [
                    isEnvDevelopment
                        ? {
                              loader: require.resolve("style-loader"),
                              options: {},
                          }
                        : { loader: MiniCssExtractPlugin.loader },
                    {
                        loader: require.resolve("css-loader"),
                        options: { sourceMap: shouldUseSourceMap },
                    },
                    {
                        loader: require.resolve("postcss-loader"),
                        options: {
                            postcssOptions: {
                                plugins: [
                                    require("postcss-flexbugs-fixes"),
                                    require("postcss-preset-env")({
                                        autoprefixer: { flexbox: "no-2009" },
                                        stage: 3,
                                    }),
                                    postcssNormalize(),
                                ],
                            },
                            sourceMap: shouldUseSourceMap,
                        },
                    },
                    {
                        loader: require.resolve("fast-sass-loader"),
                        options: {},
                    },
                ],
            },
            {
                oneOf: [
                    {
                        test: [
                            /\.bmp$/,
                            /\.gif$/,
                            /\.jpe?g$/,
                            /\.png$/,
                            /\.svg$/,
                        ],
                        loader: require.resolve("url-loader"),
                        options: {
                            limit: 10000,
                            name: isEnvDevelopment
                                ? "[path][name].[ext]"
                                : "[name].[contenthash].[ext]",
                        },
                    },

                    {
                        loader: require.resolve("file-loader"),
                        exclude: [
                            /\.vue$/,
                            /\.(js|mjs|jsx|ts|tsx)$/,
                            /\.html$/,
                            /\.json$/,
                            /\.(css|scss|sass|less)$/,
                        ],
                        options: {
                            name: isEnvDevelopment
                                ? "[path][name].[ext]"
                                : "[name].[contenthash].[ext]",
                        },
                    },
                ],
            },
            {
                test: /\.(js|mjs|jsx|ts|tsx)$/,
                type: "javascript/auto",
                loader: require.resolve("babel-loader"),
                options: {
                    sourceMaps: shouldUseSourceMap,
                    presets: [require.resolve("babel-preset-react-app")],
                    babelrc: false,
                    configFile: false,
                    customize: require.resolve(
                        "babel-preset-react-app/webpack-overrides"
                    ),
                    cacheDirectory: !0,
                    cacheCompression: isEnvProduction,
                    compact: isEnvProduction,
                },
                include: [path.resolve(__dirname, "src")],
                exclude: [/node_modules/],
            },
            {
                test: /\.(js|mjs)$/,
                exclude: [
                    /@babel(?:\/|\\{1,2})runtime/,
                    path.resolve(__dirname, "src"),
                ],
                type: "javascript/auto",
                loader: require.resolve("babel-loader"),
                options: {
                    babelrc: !1,
                    configFile: !1,
                    compact: !1,
                    presets: [
                        [
                            require.resolve(
                                "babel-preset-react-app/dependencies"
                            ),
                            { helpers: !0 },
                        ],
                    ],
                    cacheDirectory: !0,
                    cacheCompression: isEnvProduction,
                    sourceMaps: shouldUseSourceMap,
                },
                include: [/node_modules/],
            },
            {
                test: /\.(js|mjs|jsx|ts|tsx)$/,
                type: "javascript/auto",
                loader: require.resolve("babel-loader"),
                options: {
                    sourceMaps: shouldUseSourceMap,
                    plugins: [
                        [
                            require.resolve("babel-plugin-htm"),
                            {
                                pragma: "h",
                                tag: "html",
                                useBuiltIns: true,
                                useNativeSpread: true,
                            },
                        ],
                    ].filter(Boolean),
                    presets: [
                         isDevelopment && require.resolve("react-refresh/babel"),
                        require.resolve("@babel/preset-react"),

                        require.resolve("@babel/preset-typescript"),
                    ].filter(Boolean),
                    babelrc: false,
                    configFile: false,
                    cacheDirectory: !0,
                    cacheCompression: isEnvProduction,
                    compact: isEnvProduction,
                },
                include: [path.resolve(__dirname, "src")],
                exclude: [/node_modules/],
            },
            {
                test: /\.tsx?$/,
                exclude: /node_modules/,
                use: [
                    {
                        loader: require.resolve("ts-loader"),
                        options: {
                             getCustomTransformers: () => ({
                                 before: isDevelopment
                                    ? [ReactRefreshTypeScript()]
                                    : [],
                             }),
                            transpileOnly: true,
                            appendTsSuffixTo: [/\.vue$/],
                        },
                    },
                ],
            },
        ],
    },
    plugins: [
         isDevelopment && new ReactRefreshWebpackPlugin(),
        new ForkTsCheckerWebpackPlugin(),
        isEnvProduction && new CleanWebpackPlugin({ verbose: true }),
        new CopyPlugin({
            patterns: [
                {
                    globOptions: { ignore: ["**/index.html"] },
                    from: path.join(__dirname, "public"),
                    toType: "dir",
                    to: path.join(__dirname, "dist"),
                    filter: (resourcePath) => {
                        console.log("copy-webpack-plugin", resourcePath);
                        return true;
                    },
                },
            ],
        }),
        isEnvDevelopment && new webpack.HotModuleReplacementPlugin(),
        isEnvProduction &&
            new WorkboxWebpackPlugin.GenerateSW({
                skipWaiting: true,
                clientsClaim: !0,
                runtimeCaching: [
                    {
                        urlPattern: /.*\.(?:js|html|\/)$/,
                        handler: "NetworkFirst",
                        options: {},
                    },
                    {
                        urlPattern: /.*\.(?:xml|json|md|css)$/,
                        handler: "StaleWhileRevalidate",
                        options: {},
                    },
                    {
                        urlPattern: /\.(?:png|jpg|jpeg|svg|gif)$/,
                        handler: "CacheFirst",
                        options: {
                            cacheName: "image-cache",
                            expiration: { maxEntries: 10 },
                        },
                    },
                ],
            }),
        new MiniCssExtractPlugin({
            filename: "[name].[contenthash].css",
            chunkFilename: "[name].[contenthash].css",
        }),
        new VueLoaderPlugin(),
        new HtmlWebpackPlugin({
            hash: false,
            filename: "index.html",
            title: " Progressive Web Application",
            inject: !0,
            minify: {
                removeComments: !0,
                collapseWhitespace: !0,
                removeRedundantAttributes: !0,
                useShortDoctype: !0,
                removeEmptyAttributes: !0,
                removeStyleLinkTypeAttributes: !0,
                keepClosingSlash: !0,
                minifyJS: !0,
                minifyCSS: !0,
                minifyURLs: !0,
                removeAttributeQuotes: !1,
            },
            template: path.join(__dirname, "public", "index.html"),
        }),
        new PreloadWebpackPlugin({
            rel: "preload",
            include: "asyncChunks",
        }),
    ].filter(Boolean),
    optimization: {
        moduleIds: isEnvDevelopment ? "named" : "natural",
        chunkIds: isEnvDevelopment ? "named" : "natural",
        usedExports: true,
        runtimeChunk: "single",
        splitChunks: isEnvProduction
            ? {
                  chunks: "all",
                  minSize: 30000,
                  maxSize: 70 * 1000,
                  minChunks: 1,
                  maxAsyncRequests: 5,
                  maxInitialRequests: 5,
                  name: false,
                  usedExports: true,
                  cacheGroups: {
                      styles: {
                          name: "style",
                          test: /\.css$/,
                          chunks: "all",
                          enforce: true,
                      },
                  },
              }
            : {
                  chunks: "all",
                  maxSize: 20 * 1000,
              },
        minimize: isEnvProduction,
        minimizer: [
            new TerserPlugin({
                terserOptions: {
                    ecma: 5,
                    parse: { ecma: 8 },
                    compress: {
                        warnings: !1,
                        comparisons: !1,
                        inline: 2,
                        drop_console: true,
                        drop_debugger: true,
                        pure_funcs: ["console.log"],
                    },
                    mangle: { safari10: !0 },
                    output: {
                        ecma: 5,
                        comments: !1,
                        ascii_only: !0,
                    },
                },
                parallel: !0,
            }),
            new OptimizeCSSAssetsPlugin({
                cssProcessorPluginOptions: {
                    preset: [
                        "default",
                        {
                            discardComments: {
                                removeAll: true,
                            },
                            normalizeUnicode: false,
                        },
                    ],
                },
                canPrint: true,
                cssProcessorOptions: {
                    parser: safePostCssParser,
                    map: !!shouldUseSourceMap && {
                        inline: !1,
                        annotation: !0,
                    },
                },
            }),
        ],
    },
};

```
```json
{
    "name": "@masx200/webpack-react-vue-spa-awesome-config",
    "types": "./release/config/webpack.config.d.ts",
    "type": "commonjs",
    "version": "3.3.8",
    "publishConfig": {
        "registry": "https://registry.npmjs.org/",
        "access": "public"
    },
    "bin": "./bin/index.js",
    "main": "./release/config/webpack.config.js",
    "scripts": {
        "start": "webpack serve  --config ./release/config/webpack.config.js  --mode=development --port=11111",
        "build": "webpack  --config ./release/config/webpack.config.js   --mode=production",
        "rollup": "rollup -c rollup.config.js",
        "format": "prettier --write **/*.ts *.md *.json **/*.js   *.js *.ts ./cli/src/*.js ./release/config/*.js **/*.css **/*.html */*/*.js",
        "test1": "node ./bin/index.js start --port=11111",
        "test2": "node ./bin/index.js --mode=development --port=11111",
        "test3": "node ./bin/index.js build",
        "test4": "node ./bin/index.js  --mode=production",
        "test5": "webpack   --config ./release/config/webpack.config.js  --mode=development",
        "test6": "node ./bin/index.js "
    },
    "license": "MIT",
    "browserslist": {
        "production": [
            ">0.2%",
            "not dead",
            "not op_mini all"
        ],
        "development": [
            "last 1 chrome version",
            "last 1 firefox version",
            "last 1 safari version"
        ]
    },
    "dependencies": {
        "@babel/core": "^7.14.6",
        "@babel/plugin-proposal-class-properties": "^7.13.0",
        "@babel/plugin-proposal-decorators": "^7.13.15",
        "@babel/plugin-syntax-dynamic-import": "^7.8.3",
        "@babel/preset-env": "^7.14.1",
        "@babel/preset-react": "^7.13.13",
        "@babel/runtime": "^7.14.0",
        "@pmmmwh/react-refresh-webpack-plugin": "0.5.0-rc.2",
        "@types/html-webpack-plugin": "^3.2.5",
        "@types/mini-css-extract-plugin": "^1.4.3",
        "@types/node": "^15.0.2",
        "@types/optimize-css-assets-webpack-plugin": "^5.0.3",
        "@types/react": "^17.0.5",
        "@types/terser-webpack-plugin": "^5.0.3",
        "@types/webpack": "^5.28.0",
        "@types/workbox-webpack-plugin": "^5.1.6",
        "@vue/compiler-sfc": "^3.1.4",
        "@vue/preload-webpack-plugin": "^2.0.0",
        "babel-loader": "^8.2.2",
        "babel-plugin-htm": "^3.0.0",
        "babel-plugin-named-asset-import": "^0.3.7",
        "babel-plugin-transform-runtime": "^6.23.0",
        "babel-preset-react-app": "^10.0.0",
        "banner-content-loader": "^0.1.1",
        "browserslist": "4.16.6",
        "clean-webpack-plugin": "^4.0.0-alpha.0",
        "copy-webpack-plugin": "^8.1.1",
        "css-loader": "^5.2.4",
        "cssnano": "^5.0.6",
        "fast-sass-loader": "^2.0.0",
        "file-loader": "^6.2.0",
        "fork-ts-checker-webpack-plugin": "^6.2.10",
        "htm": "^3.0.4",
        "html-webpack-plugin": "^5.3.1",
        "json-loader": "^0.5.7",
        "less": "^4.1.1",
        "less-loader": "^8.1.1",
        "mini-css-extract-plugin": "^1.6.0",
        "node-sass": "^6.0.0",
        "optimize-css-assets-webpack-plugin": "^5.0.4",
        "postcss": "8.2.15",
        "postcss-flexbugs-fixes": "^5.0.2",
        "postcss-loader": "^5.2.0",
        "postcss-normalize": "^10.0.0",
        "postcss-preset-env": "^6.7.0",
        "postcss-safe-parser": "^5.0.2",
        "purgecss-webpack-plugin": "4.0.3",
        "purify-css": "^1.2.5",
        "purifycss-webpack": "^0.7.0",
        "react-app-polyfill": "^2.0.0",
        "react-refresh": "^0.10.0",
        "react-refresh-typescript": "^2.0.2",
        "regenerator-runtime": "^0.13.7",
        "sass": "1.32.12",
        "style-loader": "^2.0.0",
        "terser": "^5.7.0",
        "terser-webpack-plugin": "^5.1.4",
        "to-string-loader": "^1.1.6",
        "ts-loader": "^9.2.3",
        "unicode-loader": "^1.0.7",
        "url-loader": "^4.1.1",
        "vue-loader": "npm:vue-loader@15",
        "vue-loader-16": "npm:vue-loader@16",
        "vue-style-loader": "^4.1.3",
        "vue-template-compiler": "^2.6.14",
        "webpack": "^5.44.0",
        "webpack-cli": "^4.7.2",
        "webpack-dev-server": "^3.11.2",
        "workbox-webpack-plugin": "^6.1.5",
        "worker-loader": "3.0.8"
    },
    "devDependencies": {
        "@rollup/plugin-babel": "5.3.0",
        "@rollup/plugin-commonjs": "^19.0.0",
        "@rollup/plugin-json": "^4.1.0",
        "@rollup/plugin-node-resolve": "^13.0.0",
        "acorn": "^8.2.4",
        "core-js-pure": "^3.12.1",
        "event-target-shim": "^6.0.2",
        "lodash": "^4.17.21",
        "minimist": "^1.2.5",
        "prettier": "^2.3.0",
        "react": "^17.0.2",
        "react-dom": "^17.0.2",
        "rollup": "^2.53.1",
        "rollup-plugin-terser": "^7.0.2",
        "rollup-plugin-ts": "^1.4.0",
        "tslib": "2.2.0",
        "typescript": "^4.3.5",
        "vue": "^2.6.14",
        "yargs-parser": "20.2.7"
    },
    "repository": "https://github.com/masx200/webpack-react-vue-spa-awesome-config.git",
    "author": "masx200 <34191203+masx200@users.noreply.github.com>",
    "description": "",
    "peerDependencies": {
        "react": "17.0.2",
        "vue": "2.6.14"
    },
    "optionalDependencies": {
        "react": "17.0.2",
        "vue": "2.6.14"
    }
}

```

**Comment by Reptarsrage** [comment]
I'm still seeing this issue using v0.5.0-rc.4, does this version include the above fix?

**Comment by pmmmwh** [comment]
There's one more fix pending, sorry.

**Comment by pmmmwh** [comment]
Should be fixed in `0.5.0-rc.5`.

**Comment by Reptarsrage** [comment]
Yeah I just confirmed the issue is resolved as of version `0.5.0-rc.5`. Thanks @pmmmwh for the hard work!
